content/chronograf/v1.9/administration/managing-security.md

```diff
@@ -31,6 +31,17 @@ Configure Chronograf to use an OAuth 2.0 provider and JWT (JSON Web Token) to au
 
 (For more details on OAuth and JWT, see [RFC 6749](https://tools.ietf.org/html/rfc6749) and [RFC 7519](https://tools.ietf.org/html/rfc7519).)
 
+{{% note %}}
+#### OAuth PKCE 
+OAuth configurations in **Chronograf 1.9+** use [OAuth PKCE](https://oauth.net/2/pkce/)
+integrations for a more secure OAuth token exchange.
```

Comment:
"more secure" and "less secure" don't carry much meaning with out more context here. Perhaps we add a summary sentence about what PKCE does?